components/org.wso2.carbon.identity.oauth.uma.permission.endpoint/src/test/java/org/wso2/carbon/identity/oauth/uma/permission/endpoint/PermissionApiServiceImplTest.java

```diff
@@ -115,25 +122,29 @@ public void testRegisterPermission() throws Exception {
         when(mockHTTPServletRequest.getAttribute(anyString())).thenReturn(mockAuthenticationContext);
         String[] tokenScopes = new String[]{PAT_SCOPE};
         when(mockAuthenticationContext.getParameter(anyString())).thenReturn(tokenScopes);
-        when(mockAuthenticationContext.getUser()).thenReturn(new User());
+        when(mockAuthenticationContext.getUser()).thenReturn(mockUser);
+        when(mockUser.getUserName()).thenReturn("PRIMARY/admin");
```

Comment:
Move "PRIMARY/admin" to a constant and reuse in other places

Reply:
fixed: https://github.com/wso2-extensions/identity-oauth-uma/pull/68